### PR TITLE
Update documentation to improve visibility and avoid duplicates with Grafana documentation

### PR DESCRIPTION
**Pull request by AgnesToulet**
This PR does the following documentation update:
- Move the content of the `remote_rendering_using_docker.md` file to improve its visibility (mainly in the new image rendering configuration section in Grafana documentation).
- Move some installation instructions from Grafana documentation to the plugin main README.md (so it appears here: https://grafana.com/grafana/plugins/grafana-image-renderer/).

To be merged with this OSS PR: https://github.com/grafana/grafana/pull/38759

### CODE REVIEW

**Review by pianohacker — APPROVED** [review]
Looks good to me.

**Review by mjseaman — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -15,9 +15,15 @@ This means that you don't need to have Node.js and Chromium installed in your sy
...
-## Installation
+### Memory requirements
+
+Minimum free memory recommendation is 16GB on the system doing the rendering.
+
+Rendering images can require a lot of memory, mainly because Grafana creates browser instances in the background for the actual rendering. If multiple images are rendered in parallel, then the rendering has a bigger memory footprint. One advantage of using the remote rendering service is that the rendering will be done on the remote system, so your local system resources will not be affected by rendering.
+
+## Plugin installation
 
 ### Using grafana-cli
```
Comment (achatterjee-grafana) [inline README.md:28]:
```suggestion
### Grafana CLI
```

File: README.md [hunk 2]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
 
 ### Install in Grafana Docker image
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply README.md:36]: Please remove note about Grafana v6.4, it is a very old version.

File: README.md [hunk 3]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
 
-This plugin is not compatible with the current Grafana Docker image without installing further system-level dependencies. We recommend setting up another Docker container
-for rendering and using remote rendering, see [Remote Rendering Using Docker](#remote-rendering-using-docker) for reference.
+This plugin is not compatible with the current Grafana Docker image without installing further system-level dependencies. We recommend setting up another Docker container for rendering and using remote rendering, see [Run in Docker](#run-in-docker) for reference.
```
Comment (achatterjee-grafana) [inline README.md:38]:
```suggestion
This plugin is not compatible with the current Grafana Docker image and requires additional system-level dependencies. We recommend setting up another Docker container for rendering and using remote rendering instead. For instruction, refer to [Run in Docker](#run-in-docker). 
```

File: README.md [hunk 4]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
 
 If you still want to install the plugin in the Grafana docker image we provide instructions for how to build a custom Grafana image, see [Grafana Docker documentation](https://grafana.com/docs/installation/docker/#custom-image-with-grafana-image-renderer-plugin-pre-installed) for further instructions.
```
Comment (achatterjee-grafana) [inline README.md:40]:
```suggestion
If you still want to install the plugin with the Grafana Docker image, refer to the instructions on building a custom Grafana image in [Grafana Docker documentation](https://grafana.com/docs/installation/docker/#custom-image-with-grafana-image-renderer-plugin-pre-installed).
```

File: README.md [hunk 5]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
+This plugin can also be run as a remote HTTP rendering service. In this setup, Grafana renders an image by making a HTTP request to the remote rendering service, which in turn renders the image and returns it back in the HTTP response to Grafana.
 
-## Remote Rendering Using Docker
+You can run the remote HTTP rendering service using Docker or as a standalone Node.js application.
 
-Instead of installing and running the image renderer as a plugin, you can run it as a remote image rendering service using Docker. Read more about [remote rendering using Docker](https://github.com/grafana/grafana-image-renderer/blob/master/docs/remote_rendering_using_docker.md).
+### Run in Docker
 
-## Troubleshooting
+The docker images are published at [Docker Hub](https://hub.docker.com/r/grafana/grafana-image-renderer).
```
Comment (achatterjee-grafana) [inline README.md:52]:
```suggestion
Grafana Docker images are published at [Docker Hub](https://hub.docker.com/r/grafana/grafana-image-renderer).
```

File: README.md [hunk 6]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
+
+The following example shows how to run Grafana and the remote HTTP rendering service in two separate Docker containers using Docker Compose.
```
Comment (achatterjee-grafana) [inline README.md:54]:
```suggestion
The following example shows how you can run Grafana and the remote HTTP rendering service in two separate Docker containers using Docker Compose.
```

File: README.md [hunk 7]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
+
+Create a `docker-compose.yml` with the following content:
```
Comment (achatterjee-grafana) [inline README.md:56]:
```suggestion
1. Create a `docker-compose.yml` with the following content:
```

File: README.md [hunk 8]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
+      GF_RENDERING_SERVER_URL: http://renderer:8081/render
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000/
+      GF_LOG_FILTERS: rendering:debug
+  renderer:
+    image: grafana/grafana-image-renderer:latest
+    ports:
+      - 8081
+```
+
+And then run:
```
Comment (achatterjee-grafana) [inline README.md:76]:
```suggestion
1. Next, run docker compose.
```

File: README.md [hunk 9]
```diff
@@ -29,25 +35,83 @@ grafana-cli plugins install grafana-image-renderer
...
+1. Restart Grafana.
+
+## Configuration
+
+For available configuration settings, please refer to [Grafana Image Rendering documentation](https://grafana.com/docs/image_rendering/#configuration).
+
+## Troubleshooting
 
-See [docs](https://github.com/grafana/grafana-image-renderer/blob/master/docs/index.md).
+For troubleshooting help, please refer to
```
Comment (achatterjee-grafana) [inline README.md:116]:
```suggestion
For troubleshooting help, refer to
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: README.md [hunk 10]
```diff
@@ -15,9 +15,15 @@ This means that you don't need to have Node.js and Chromium installed in your sy
 
 However, [Chromium browser](https://www.chromium.org/) depends on certain libraries and if you don't have all of those libraries installed in your
 system you may encounter errors when trying to render an image. For further details and troubleshooting help, please refer to
-[Grafana Image Rendering documentation](https://grafana.com/docs/administration/image_rendering/).
+[Grafana Image Rendering documentation](https://grafana.com/docs/image_rendering/).
```
Comment (AgnesToulet) [inline README.md:18]:
```suggestion
[Grafana Image Rendering documentation](https://grafana.com/docs/image-rendering/).
```

File: README.md [hunk 11]
```diff
@@ -15,9 +15,15 @@ This means that you don't need to have Node.js and Chromium installed in your sy
...
 
-## Installation
+### Memory requirements
```
[the remark this answers is not in the document]
Comment (AgnesToulet) [reply README.md:20]: The top section is already "Requirements", you can't see it in the PR review tab as it already existed and I didn't make any change to this top section. You can see it here: https://github.com/grafana/grafana-image-renderer#requirements

File: README.md [hunk 12]
```diff
@@ -15,9 +15,15 @@ This means that you don't need to have Node.js and Chromium installed in your sy
...
+
+Minimum free memory recommendation is 16GB on the system doing the rendering.
+
+Rendering images can require a lot of memory, mainly because Grafana creates browser instances in the background for the actual rendering. If multiple images are rendered in parallel, then the rendering has a bigger memory footprint. One advantage of using the remote rendering service is that the rendering will be done on the remote system, so your local system resources will not be affected by rendering.
+
+## Plugin installation
```
[the remark this answers is not in the document]
Comment (AgnesToulet) [reply README.md:26]: Applied this with some updates as I had trouble understanding the last paragraph.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AgnesToulet) [reply README.md:36]: Done.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AgnesToulet) [reply README.md:28]: Done, with an update to point to the recommended way to do it.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM, if we need any additional change, we can do it later. Thank you!

**Comment by AgnesToulet** [comment]
Great, thanks! I'll wait for the Grafana PR to be approved to merge both PRs around the same time (as some contents were moved between the two docs and there are a lot of links between these docs).

**Pull request merged**